tests/unit/test_iop.py

```diff
@@ -209,7 +209,7 @@ def test_not_published_record():
     """
     response = fake_response_from_string(body)
     node = get_node(spider, "Article", response)
-    spider.pdf_files = get_responses_path('iop', 'pdf')
+    spider.pdf_files = get_test_suite_path('unit', 'responses', 'iop', 'pdf')
```

Comment:
Tip: if you put 'unit' as the default value for the `test_suite` parameter, then you can skip it almost everywhere.